__tests__/schema/threads.ts

```diff
@@ -0,0 +1,90 @@
+import { gql } from 'apollo-server'
+import { rest } from 'msw'
+
+import { article, user } from '../../__fixtures__/uuid'
+import { comment1, comment2, comment3 } from '../../__fixtures__/uuid/comment'
+import { Service } from '../../src/graphql/schema/types'
+import { UuidPayload } from '../../src/graphql/schema/uuid/abstract-uuid'
+import { CommentPayload } from '../../src/graphql/schema/uuid/comment'
+import { Instance } from '../../src/types'
+import {
+  assertSuccessfulGraphQLQuery,
+  Client,
+  createTestClient,
+  createThreadsHandler,
+} from '../__utils__'
+
+let client: Client
+
+beforeEach(() => {
+  client = createTestClient({
+    service: Service.SerloCloudflareWorker,
+    user: user.id,
+  }).client
+})
+
+// This test does not make sense
+test('Threads', async () => {
+  setupThreads(article, [[comment1, comment2], [comment3]])
+  await assertSuccessfulGraphQLQuery({
+    query: gql`
+      query uuid($id: Int!) {
+        uuid(id: $id) {
+          __typename
+          #threads {
+          #totalCount
+          #nodes {
+          #createdAt
+          #updatedAt
+          #title
+          #archived
+          #trashed
+          #}
+          #}
+        }
+      }
+    `,
+    variables: { id: article.id },
+    data: {
+      uuid: article,
+    },
+    client,
+  })
+})
+
+function setupThreads(uuidPayload: UuidPayload, threads: CommentPayload[][]) {
+  global.server.use(
+    createThreadsHandler(
+      uuidPayload.id,
+      threads.map((thread) => thread[0].id)
+    )
+  )
+  global.server.use(
+    rest.get(
+      `http://${Instance.De}.${process.env.SERLO_ORG_HOST}/api/uuid/${uuidPayload.id}`,
+      (req, res, ctx) => {
+        //Findet keine Id hier!!
+        const id = Number(req.params.id)
+        const thread = threads.find((thread) =>
+          thread.find((comment) => comment.id === id)
+        )
+        if (thread === null || thread === undefined) {
```

Comment:
```suggestion
        if (thread === undefined) {
```

From the documentation: "The value of the first element in the array that satisfies the provided testing function. Otherwise, undefined is returned." ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find )

Or: `if (R.isNil(thread))`